### ISSUE TITLE
## nested-interactive AXE a11y errors violations

### ISSUE THREAD

**Issue by qunabu** [EscolaLMS/Components#248]
## AXE a11y errors violations
    
1 accessibility violations were detected

| id                 | impact  | description                                                                                                                                           | nodes |
| ------------------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- | ----- |
| nested-interactive | serious | Ensures interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies | 8     |
    

_Originally posted by @github-actions[bot] in https://github.com/EscolaLMS/Components/issues/247#issuecomment-1282447802_
